src/database/DataTypes/TransactionItem.js

```diff
@@ -48,7 +48,7 @@ export class TransactionItem extends Realm.Object {
         this.transaction.isFinalised)) {
       return this.item.totalQuantity + this.totalQuantity;
     }
-    return this.item.totalQuantity;
+    return this.item.totalQuantity;  // TODO think about this logic, will never hit this return, createRecord, new invoice is always confirmed
```

Comment:
Ahh that's true.  It's not used in SupplierInvoice currently, I can see it could be.